db/migrate/20180813141443_create_ckeditor_assets.rb

```diff
@@ -0,0 +1,23 @@
+class CreateCkeditorAssets < ActiveRecord::Migration
+  def self.up
+    create_table :ckeditor_assets do |t|
+      t.string  :data_file_name, null: false
+      t.string  :data_content_type
+      t.integer :data_file_size
+      t.string  :data_fingerprint
+      t.string  :type, limit: 30
+
+      # Uncomment	it to save images dimensions, if your need it
```

Comment:
Typo: you*

But the bigger questions here are the following:
* What should we uncomment? Are you referring to the 2 next lines for `width` and `integer`?
* Wouldn't it be better to allow those fields to be empty and then handle through the model if we want to save those values?